Traceback (most recent call last):
  File "server.py", line 410, in <module>
    asyncio.run(bootstrap()); asyncio.run(main())
  File "C:\Program Files\python\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Program Files\python\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "server.py", line 398, in main
    async with websockets.serve(serve_socket, HOST, PORT, ping_interval=HEARTBEAT_SEC, ping_timeout=TIMEOUT_SEC):
  File "E:\python\socp\.venv\lib\site-packages\websockets\legacy\server.py", line 1098, in __aenter__
    return await self
  File "E:\python\socp\.venv\lib\site-packages\websockets\legacy\server.py", line 1116, in __await_impl__
    server = await self._create_server()
  File "C:\Program Files\python\lib\asyncio\base_events.py", line 1463, in create_server
    raise OSError(err.errno, 'error while attempting '
OSError: [Errno 10048] error while attempting to bind on address ('0.0.0.0', 9001): 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
